server/views/applications/new.njk

```diff
@@ -13,8 +14,30 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Applications</h1>
+
+        <h2>Existing applications</h2>
+            {{
+              govukTable({
+                  caption: title,
+                  captionClasses: "govuk-table__caption--m",
+                  firstCellIsHeader: true,
+                  head: [
+                    {
+                      text: "Person"
+                    },
+                    {
+                      text: "CRN"
+                    }
+                  ],
+                  rows: dashboardTableRows(applications.applications)
```

Comment:
should we just pass in `applications` instead of `applications.applications`? I'm getting an error atm...